sdk/storage/azure_storage_blob/tests/blob_client.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use azure_core_test::recorded;
+use azure_identity::DefaultAzureCredentialBuilder;
+use azure_storage_blob::{BlobBlobClientGetPropertiesOptions, BlobClient, BlobClientOptions};
+use std::{env, error::Error};
+
+#[recorded::test(live)]
+async fn test_get_blob_properties() -> Result<(), Box<dyn Error>> {
+    // Setup
+    let storage_account_name = env::var("AZURE_STORAGE_ACCOUNT_NAME")
+        .expect("Failed to get environment variable: AZURE_STORAGE_ACCOUNT_NAME");
+    let endpoint = format!("https://{}.blob.core.windows.net/", storage_account_name);
+    let credential = DefaultAzureCredentialBuilder::default().build()?;
+
+    // Act
+    let blob_client = BlobClient::new(
+        &endpoint,
+        String::from("testcontainer"),
+        String::from("test_blob.txt"),
+        credential,
+        Some(BlobClientOptions::default()),
```

Comment:
Pass `None`, though once you use test-proxy you'll need to instrument your `BlobClientOptions::client_options` anyway, but don't show customers to do this in doc tests or examples. It's unnecessarily verbose and not idiomatic.